supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendSingletonActor.scala

```diff
@@ -1,12 +1,12 @@
 package cromwell.backend.impl.jes
 
 import akka.actor.{Actor, ActorLogging, Props}
-import cromwell.backend.impl.jes.statuspolling.{JesApiQueryManager}
+import cromwell.backend.impl.jes.statuspolling.JesApiQueryManager
 import cromwell.backend.impl.jes.statuspolling.JesApiQueryManager.DoPoll
 
-class JesBackendSingletonActor extends Actor with ActorLogging {
+final case class JesBackendSingletonActor(jesConfiguration: JesConfiguration) extends Actor with ActorLogging {
```

Comment:
ToL I've recently gone off the idea of using `case class` for actors. They aren't being used anything like ADTs and this limits their future extensibility.